### PR TITLE
Add Xenit Platform config

### PR DESCRIPTION
**Pull request by simongottschlag**

### CODE REVIEW

File: modules/kubernetes/aks-core/modules.tf [hunk 1]
```diff
@@ -310,3 +310,19 @@ module "azad_kube_proxy" {
...
+module "platform" {
+  for_each = {
+    for s in ["platform"] :
+    s => s
+    if var.platform_enabled
+  }
+
+  source = "../../kubernetes/platform"
+
+  platform_config = var.platform_config
```
Comment (phillebaba) [inline modules/kubernetes/aks-core/modules.tf:324]: Config should be used in the aks_core module but we should avoid using these types of objects in the root module.

File: modules/kubernetes/platform/platform-proxy.tf [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    value = var.platform_config.identity.tenant_id
+  }
+
+  set {
+    name  = "keyVaultName"
+    value = var.platform_config.azure_key_vault_name
+  }
+}
+
+resource "helm_release" "platform_proxy_thanos_receiver" {
```
Comment (phillebaba) [inline modules/kubernetes/platform/platform-proxy.tf:27]: This is in the platform namespaces so having platform in the name is redundant.

**Review by phillebaba — APPROVED** [review]

File: modules/kubernetes/README.md [hunk 3]
```diff
@@ -25,3 +25,12 @@ This directory contains all the Kubernetes Terraform modules.
 - [`azad-kube-proxy`](azad-kube-proxy/README.md)
 - [`prometheus`](prometheus/README.md)
 - [`ingress-healthz`](ingress-healthz/README.md)
+<<<<<<< HEAD
+- [`xenit`](xenit/README.md)
+=======
+<<<<<<< HEAD
+>>>>>>> 7c2a8e8 (Add ingress healthz (#199))
+=======
```
Comment (phillebaba) [inline modules/kubernetes/README.md:33]: Dont commit merge conflict

Comment (simongottschlag) [reply modules/kubernetes/README.md]: fixed

**Review by phillebaba — APPROVED** [review]

**Pull request merged**